### PR TITLE
Re-Revert PR 266

### PR DESCRIPTION
**Pull request by donyunardi**
Now that the release has been completed for 0.3.1, we are re-reverting PR #266

This reverts commit 74f64407cc9d1bb0ea89df3d40054bb6dd3248a1.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # teal.reporter 0.3.1.9000
 
+# teal.reporter 0.3.1
+
```
Comment (vedhav) [inline NEWS.md:4]:
```suggestion
```

**Review by vedhav — APPROVED** [review]
LGTM ~after we remove the line from `NEWS.md`~

File: NEWS.md (hunk 1 above)
Comment (vedhav) [reply NEWS.md:4]: my bad

Comment (donyunardi) [reply NEWS.md:4]: As discussed, we need this line.

**Pull request merged**